lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -682,7 +660,7 @@ export class NativeInteractionClient extends BaseInteractionClient {
             );
 
         const nativeCacheRecord = new CacheRecord(
-            accountEntity,
+            undefined,
```

Comment:
May i ask you why accountEntity is undefined?

Reply:
In the native flow we cache tokens in-memory (the native broker contains its own persistent cache). The account object, however, needs to persist in the browser. Storing the account in both in-memory and browser storage creates two sources of truth so instead, we store the account in browser storage above and then this call stores only the id and access tokens in in-memory storage

Reply:
Got it, Thanks for the detailed explanation, appreciate it!